src/ray/common/id.cc

```diff
@@ -61,7 +61,8 @@ constexpr uint8_t kCreatedByTaskBitsOffset = 15;
 constexpr uint8_t kObjectTypeBitsOffset = 14;
 
 /// The bit offset of the flag `TransportType` in a flags bytes.
-constexpr uint8_t kTransportTypeBitsOffset = 11;
+// TODO(edoakes): this isn't used anymore.
+// constexpr uint8_t kTransportTypeBitsOffset = 11;
```

Comment:
Remove?

Reply:
Leaving as a reminder to update the ID spec.